src/index.js

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env node --use_strict
 let path = require('path');
 let steps = require('./steps');
+let check = require('./check');
```

Comment:
Could we name this something different than check for clarity. Maybe checkVersion?